typescript/token/src/config.ts

```diff
@@ -27,9 +27,11 @@ export const isErc20Metadata = (metadata: any): metadata is ERC20Metadata =>
   metadata.decimals && isTokenMetadata(metadata);
 
 export type SyntheticConfig = TokenMetadata & {
+  votable?: boolean;
   type: TokenType.synthetic | TokenType.syntheticUri;
```

Comment:
would it be more consistent to add another `TokenType`? don't feel strongly about this one